### DISCUSSION TITLE
Why does .predict and .predict_oof return different results?

### DISCUSSION THREAD

**Discussion by Vikram12301** [autogluon/autogluon#4346] [General]
I was using .predict and .predict_oof, I expected the results for both to be same, But it was different in some cases. THe orange line is the predictions from .predict and green line has the predictions from .predict_oof
<img width="180" alt="Screenshot 2024-07-29 at 9 54 38 AM" src="https://github.com/user-attachments/assets/67468a05-fdf7-4283-b04b-6acd434887d2">

**Comment by Innixma** [comment]
They are not meant to be the same. Refer to the docs: 

https://auto.gluon.ai/stable/api/autogluon.tabular.TabularPredictor.predict_oof.html
https://auto.gluon.ai/stable/api/autogluon.tabular.TabularPredictor.predict_proba_oof.html
